client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/task-helper.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @jest-environment jsdom
+ */
+import { getArrayOfFilteredTasks, getEnhancedTasks, isTaskDisabled } from '../task-helper';
+import { tasks, launchpadFlowTasks } from '../tasks';
+import { Task } from '../types';
+
+function getTask( taskData = {} ) {
+	const task: Task = {
+		id: 'foo_task',
+		isCompleted: false,
+		actionUrl: '#',
+		taskType: 'blog',
+		displayBadge: false,
+	};
+
+	return { ...task, ...taskData };
+}
+
+describe( 'Task Helpers', () => {
+	describe( 'getEnhancedTasks', () => {
+		describe( 'when a task should not be enhanced', () => {
+			it( 'then it is not enhanced', () => {
+				const fakeTasks = [
+					getTask( { id: 'fake-task-1' } ),
+					getTask( { id: 'fake-task-2' } ),
+					getTask( { id: 'fake-task-3' } ),
+				];
+				// eslint-disable-next-line @typescript-eslint/no-empty-function
+				expect( getEnhancedTasks( fakeTasks, 'fake.wordpress.com', null, () => {} ) ).toEqual(
+					fakeTasks
+				);
+			} );
+		} );
+	} );
+
+	describe( 'getArrayOfFilteredTasks', () => {
+		describe( 'when no flow is provided', () => {
+			it( 'then no tasks are found', () => {
+				expect( getArrayOfFilteredTasks( tasks, null ) ).toBe( null );
+			} );
+		} );
+
+		describe( 'when a non-existing flow is provided', () => {
```

Comment:
It's strange that `getArrayOfFilteredTasks` returns `null` when no flow is provided and `undefined` when no tasks are found. Maybe we should consolidate the return values?

Reply:
Do you think it should be an empty array in those cases? or other thoughts?

Reply:
I like empty arrays 👍